### DISCUSSION TITLE
Picture and Avatar store on MySQL Database

### DISCUSSION THREAD

**Discussion by votrongbang** [snipe/snipe-it#15915] [Q&A] [answered]
I am using Docker Compose, and I want to store this information, snipeit:/var/lib/snipeit, in the MySQL database. Is that possible?

**Comment by rashmikanaveen** [comment, marked as answer]
Storing images and avatars directly in a MySQL database is possible, but it not recommended due to performance and scalability concerns. Instead, it's often better to store the images in a file storage system (like AWS S3, Google Cloud Storage,etc..) and store the file paths or URLs in the database.

However, if you still want to store images in MySQL, you can use the 
BLOB (Binary Large Object) data type to store binary data.
